docs/examples.md

```diff
@@ -45,6 +45,39 @@ passwd:
       shell: /bin/bash
 ```
 
+### Using Password Authentication
+
+Fedora CoreOS ships with no default passwords. You can use a Fedora CoreOS Config to set a password for a local user. Building on the previous example, we can confgure the `password_hash` for one or more users:
```

Comment:
I'm inclined to say that we shouldn't be documenting distro defaults in the tool docs, so I'd probably drop the first sentence.

spelling: `configure`

Reply:
I see your :+1: but no code change for this comment.  Did that part get missed?

Reply:
I saw the review after the push, I'll be updating it shortly. Just wanted to indicate I saw it